### DISCUSSION TITLE
尝试做了绳网的AppImage文件

### DISCUSSION THREAD

**Discussion by Anyi-qaq** [share121/inter-knot#5906] [公告]
![粘贴的图像](https://github.com/user-attachments/assets/b46b64df-c933-43fa-81dd-24fe5beb8a7c)
# 在GPT的帮助下尝试做了绳网的AppImage文件
第一次做这种尝试

**Comment by Anyi-qaq** [comment]
放在了123盘里
https://www.123865.com/s/nLezVv-hvFD3

**Reply by Anyi-qaq** [reply]
以及就是现在帖子怎么只能发投票

**Comment by Anyi-qaq** [comment]
我接下来会试着放到aur里

**Comment by coolwide** [comment]
佬
![Uploading 1723608875426.png…]()
